src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -76,6 +77,8 @@
 
   @Column private String actionPlanId;
 
+  @Column private Date createdDateTime;
```

Comment:
use LocalDateTime

Reply:
there was some discussion the other day about storing all dates in UTC, but I agree that LocalDateTime seems to be the modern way